docs/glossary.md

```diff
@@ -0,0 +1,144 @@
+# Glossary
+
+This page defines some terminology that is commonly used in Electron development.
+
+### ASAR
+
+ASAR stands for Atom Shell Archive Format. An [asar][asar] archive is a simple
+`tar`-like format that concatenates files into a single file. Electron can read
+arbitrary files from it without unpacking the whole file.
+
+The ASAR format was created primarily to improve performance on Windows... TODO
+
+### Brightray
+
+Brightray is a static library that makes [libchromiumcontent] easier to use in
+applications. It was created specifically for Electron, but can be used to
+enable Chromium's renderer in native apps that are not based on Electron.
+
+Brightray is a low-level dependency of Electron that does not concern the
+majority of Electron users.
+
+### DMG
+
+An Apple Disk Image is a packaging format used by macOS. DMG files are
+commonly used for distributing application "installers". [electron-builder]
+supports `dmg` as a build target.
+
+### IPC
+
+IPC stands for Inter-Process Communication. Electron uses IPC to send
+serialized JSON messages between the [main] and [renderer] processes.
+
+### libchromiumcontent
+
+A single, shared library that includes the Chromium Content module and all its
+dependencies (e.g., Blink, [V8], etc.).
+
+### main process
```

Comment:
I'm not like super strong about it, just trying to group related stuff together.